core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3756,6 +3757,7 @@ private void startSyncAndJournal(RpcContext rpcContext, AlluxioURI uri)
       try {
         mSyncManager.startSyncPostJournal(uri);
       } catch (Throwable e) {
+        LOG.info("Start sync failed because {}", ExceptionUtils.getStackTrace(e));
```

Comment:
This should not be an `info` log, right? Also, will this be spammy?

Reply:
this should not be spammy because startSync should rarely fail.
what level do you think is appropriate?  
I was actually debating between `warn` and `debug`, and eventually settled on `info`